docker/worker-base/build.sh

```diff
@@ -18,6 +18,6 @@ cd ../../
 
 PROJECT=${GOOGLE_CLOUD_PROJECT:-oss-vdb}
 
-docker build -t gcr.io/$PROJECT/worker-base:$1 -t gcr.io/$PROJECT/worker-base:latest -f docker/worker-base/Dockerfile . && \
+docker build -t gcr.io/$PROJECT/worker-base:$1 -t gcr.io/$PROJECT/worker-base:latest -f docker/worker-base/Dockerfile --pull . && \
```

Comment:
What does the `--pull` do here?

Reply:
It tells it to always grab the latest version of the base image (ubuntu:20.04), rather than using the cached one.